src/logging.h

```diff
@@ -155,12 +155,10 @@ static inline void LogPrintf(const char* fmt, const Args&... args)
     }
 }
 
-template <typename... Args>
-static inline void LogPrint(const BCLog::LogFlags& category, const Args&... args)
-{
-    if (LogAcceptCategory((category))) {
-        LogPrintf(args...);
-    }
-}
+#define LogPrint(category, ...) do { \
```

Comment:
I'd prefer to see a developer comment to explain why this is a macro (not a function like `LogPrintf` just a few lines up).

optional: You may install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to align the newline continuation chars `\`.

Reply:
Done in 8734c85.